float64[] desiredPose  
int32 finger # finger = 0 rel_indexTip_thumbTip , finger=5 for thumb, finger4=index, and so on
bool getJacobian
---
float64 normError
float64[] jointPosition
kinematics_control_msgs/singleRowColumn[] jacobian
